spec/acceptance/ntp_daemon_opts_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+config = if fact('osfamily') == 'redhat'
+           '/etc/sysconfig/ntp'
+         else
+           '/etc/default/ntp'
+         end
+
+describe 'ntp class with daemon options:', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+  context 'when successful' do
+    let(:pp) { "class { 'ntp': user => 'ntp', daemon_extra_opts => '-g -i /var/lib/ntp' }" }
+
+    it 'runs twice' do
```

Comment:
If you rebase your work we actually now have an idempotent apply method that we prefer to use for code like this. It cut's down on the overhead by only transferring the manifest once.

Reply:
You can find it in spec_helper_acceptance.rb

Reply:
Thanks!
found it https://github.com/puppetlabs/puppetlabs-ntp/commit/ef51cb4fd27ffe01bd6d71e521b1a51549111366#diff-174f7dfb508155027cb4c3e9a1840591
but does not seem used for now or no example :)

found one here
https://github.com/puppetlabs/puppetlabs-mysql/blob/ea12c2ea0aa966e441a823c187837d94f1523ac0/spec/acceptance/types/mysql_database_spec.rb#L46
so just moving to
```
    it 'run module twice idempotently' do
      idempotent_apply(default, pp, {})
   end
```
right?

Reply:
Yep, look's good.